ServerCore/Pages/Resources/PH21New/RulesPartial.cshtml

```diff
@@ -7,15 +7,24 @@
         <li id="WhatToBring">
             The more prepared you are for Puzzlehunt weekend, the better time you'll have participating in it. Here's a list of supplies that you may find helpful:
             <ul>
-                <li>
-                    Office supplies: Pens, pencils, colored pencils, graph paper, scissors,
-                    clear tape, ruler/straight edge, protractor, highlighters, glue stick, compass
+                <li>Most puzzles can be solved without printing, but a few will require a printer (ideally color), as well as scissors and tape.</li>
```

Comment:
I thought you said you fixed mentioning tape and scissors twice 😛

Reply:
:p - fixed now, really.